pkg/process/checks/process.go

```diff
@@ -254,8 +256,10 @@ func fmtProcesses(
 	ctrByProc map[int32]string,
 	syst2, syst1 cpu.TimesStat,
 	lastRun time.Time,
+	connsByPID map[int32][]*model.Connection,
 ) map[string][]*model.Process {
 	procsByCtr := make(map[string][]*model.Process)
+	connCheckIntervalS := int(cfg.CheckIntervals[config.ConnectionsCheckName] / time.Second)
```

Comment:
Minor: since this is always cast to `float32` later on, should we just make it `float32` here?

Reply:
Technically that's correct, but I feel a bit weird using float to represent seconds. Since the conversion is cheap I might just leave this one.